andela/ah-the-immortals-backend
# #165305765 Implement user notifications

### What does this PR do?
- Implement user notifications
### What are the tasks to be completed?
* Implement email and in app notifications
### How can this be manually tested?
### Local Development Setup
Refer to the project Readme.md for this.
**Test with postman**
1. Signing up two users and verify
2. Follow one user: 
```
POST  http://127.0.0.1:8000/api/profiles/<*username*>/follow/
```
3. Create an article using the followed user:
```
POST http://127.0.0.1:8000/api/articles/
```
4. Create a comment with the followed user:
```
POST http://127.0.0.1:8000/api/articles/<*slug*>/comments/
```
5. Fetch the following-user notifications using:
```
GET http://127.0.0.1:8000/api/notifications/
```
**Screenshots**
![image](https://user-images.githubusercontent.com/41931013/57491294-f7ea6180-72c4-11e9-923d-d912e5f37a54.png)
### What are the relevant pivotal tracker stories?
[#165305765(https://www.pivotaltracker.com/n/projects/2326460/stories/165305765)
